src/client/menu/menu.c

```diff
@@ -2365,6 +2373,13 @@ Options_MenuInit(void)
     s_options_quality_list.generic.callback = UpdateSoundQualityFunc;
     s_options_quality_list.itemnames = quality_items;
 
+    s_options_audiopause_box.generic.type = MTYPE_SPINCONTROL;
+    s_options_audiopause_box.generic.x = 0;
+    s_options_audiopause_box.generic.y = (y += 10);
+    s_options_audiopause_box.generic.name = "pause sound on pause";
```

Comment:
This overflows the menu on 320x280. And do we really want to use the last available line in the options menu for a rather obscure feature like this? I think having it as cvar only is good enough.